include/types/Profiles.h

```diff
@@ -74,6 +74,26 @@ class Profiles : public PerformanceTable {
    */
   void generateRandomPerfValues(unsigned long int seed = time(NULL),
                                 int lower_bound = 0, int upper_bound = 1);
+
+  /**
+   * getBelowAndAboveProfile gets the profiles below and above the given
+   * profile. If given profile is the first or last, it will return itself
+   *
+   * @param profId id of the profile to get the above and below profiles
+   *
+   * @return profiles performances (below and above)
+   */
+  std::pair<std::vector<Perf>, std::vector<Perf>>
+  getBelowAndAboveProfile(std::string profName);
+
+  /**
+   * setPerf set a Perf given the alt name, crit and value
+   *
+   * @param name name of the alt or profile we want to update
+   * @param crit name of the crit we want to update
+   * @param value new perf value
+   */
+  void setPerf(std::string name, std::string crit, float value);
```

Comment:
Should we better see those types of functions be put in PerformanceTable than Profiles ?

Reply:
I actually prefer it in here instead of in PerformanceTable, but I'm open to discussion! The reason is, setPerf is highly uneficient and we should avoir using it. However - correct me if I'm wrong - Profiles object should remain quite small in size, in comparison to datasets. Therefore, only defining it for Profiles prevent its use for datasets. 

Anyway, it could be great to add a comment on the doc that this is eneficient and shouldn't be use often

Reply:
sounds good